src/SalmonDE/Updater/UpdaterTask.php

```diff
@@ -24,8 +24,8 @@ public function onRun($currenttick){
                 foreach(glob("plugins/*".$this->name."*.phar") as $phar){
                     unlink($phar);
                 }
-                file_put_contents('plugins/'.$this->name.'.phar', $file);
-                if(!file_exists('plugins/'.$this->name.'.phar')){
+                file_put_contents('plugins/'.$this->name.' v'.$this->newversion.'.phar', $file);
+                if(!file_exists('plugins/'.$this->name.' v'.$this->newversion.'.phar')){
```

Comment:
@opkiler22789 hm wait, I think it's better to make them not havve the version in the name. Just to prevent that the plugin exists multiple times